apps/frontpage/components/home/illustrated-feature-list.tsx

```diff
@@ -70,14 +70,14 @@ export const IllustratedFeatureList: FC<IllustratedFeatureListProps> = ({
           'rounded-lg relative overflow-hidden hidden h-full min-h-[640px] md:block lg:max-h-[640px] min-[1416px]:ml-0 min-[1416px]:mr-0 min-[1416px]:rounded-lg',
           alignment === 'left'
             ? 'order-1 md:-ml-12 md:rounded-tl-none md:-border-bl-none'
-            : 'order-2 md:-mr-12 md:rounded-tr-none md:-border-br-none'
+            : 'order-2 md:-mr-12 md:rounded-tr-none md:-border-br-none',
         )}
         style={{ backgroundColor: bgColor }}
       >
         <AnimatePresence initial={false} custom={direction}>
           <motion.div
             className="absolute top-0 bottom-0 left-0 right-0"
-            key={activeFeature.title}
+            key={activeFeature?.title}
```

Comment:
Better, I think, to cast the type of `activeFeature`:

```ts
const activeFeature = features[activeIndex] as FeatureItem;
```

That assumes that `activeIndex` will always be an index in the `features` array, but that seems safe to me.

Then you can rely on `activeFeature`'s type throughout this code.